code/modules/paperwork/photocopier.dm

```diff
@@ -316,11 +316,11 @@
 		return
 	src.add_fingerprint(user)
 	if(target == user && !user.incapacitated())
-		visible_message("<span class='warning'>[usr] jumps onto the photocopier!</span>")
+		visible_message("<span class='warning'>[usr] jumps onto the [src]!</span>")
```

Comment:
`the [src]` will probably result in "the the fax machine". Make sure to test that this isn't the case.

Reply:
It will, remove all of the hardcoded `the`s please.